docs/agents/changeInterfaceColorAgent.ru.md

```diff
@@ -0,0 +1,39 @@
+Данный агент изменяет цвет компонента интерфейса.
+
+**Класс действий:**
+
+`action_change_interface`
+
+
+**Параметры:**
+
+1. `message` node связана с `interface component` с помощью отношения `rrel_entity` и при необходимости с `component color` отношением`rrel_color`.
+
+**Ход работы агента:**
+
+* Агент находит цвет компонента с помощью отношения `nrel_component_color` и меняет его цвет в соответствии с значением sc-ссылки.
+* Если конкретный цвет не передается, генерируется случайный цвет.
+* Агент вызывается логическим правилом с конкретным цветом или случайным.
+
+Правило для цвета конкретного компонента интерфейса:
+<img src="../images/lr_color_message.png"></img>
+
+Правило цвета случайного компонента интерфейса:
+<img src="../images/lr_random_color_message.png"></img>
+
+<img src="../images/interfaceComponentColorExample.png"></img>
+
+### Пример
+
+Пример входной структуры:
+
+<img src="../images/changeInterfaceColorAgentInput.png"></img>
+
+Чтобы протестировать, Вы можете использовать следующие сообщения: `Измени цвет хедера на #001122`, `Поменяй цвет компонента футер на #CC1122` или `Измени цвет компонента основная часть на #00AAFF`.
```

Comment:
I suggest to move this up even before action class for user to see how to test it